assets/css/src/components-public/_public.scss

```diff
@@ -7,6 +12,26 @@
 .mailpoet_list_label,
 .mailpoet_date_label {
   display: block;
+
+  @include breakpoint-max-width($form-break-small-width - 1) {
+    font-size: 16px !important;
```

Comment:
Shouldn't we also use a variable for those sizes? Both `16px` and `1.4`? They are used in at least two places right now.